dev/main.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* global requirejs */
+/* global define */
+
+requirejs.config({
+	paths: {
+		jquery: '//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min',
```

Comment:
I'd advise against requiing use of jquery for something that's going to get used everywhere and needs to be performant; is it strictly necessary?